packages/perseus-editor/src/widgets/interactive-graph-editor/start-coords/util.test.ts

```diff
@@ -318,12 +318,12 @@ describe("getAngleEquation", () => {
         ${[2, 1]}   | ${[2, 1]} | ${[2, 1]}                                  | ${"0° angle at (2, 1)"}
         ${[5, 0]}   | ${[0, 0]} | ${[0, 5]}                                  | ${"90° angle at (0, 0)"}
         ${[5, 0]}   | ${[0, 0]} | ${[-5, 5]}                                 | ${"135° angle at (0, 0)"}
-        ${[5, 0]}   | ${[0, 0]} | ${[-5, -5]}                                | ${"225° angle at (0, 0)"}
-        ${[5, 0]}   | ${[0, 0]} | ${[5, -5]}                                 | ${"315° angle at (0, 0)"}
-        ${[0, 5]}   | ${[0, 0]} | ${[5, 0]}                                  | ${"-90° angle at (0, 0)"}
-        ${[-5, 5]}  | ${[0, 0]} | ${[5, 0]}                                  | ${"-135° angle at (0, 0)"}
-        ${[-5, -5]} | ${[0, 0]} | ${[5, 0]}                                  | ${"-225° angle at (0, 0)"}
-        ${[5, -5]}  | ${[0, 0]} | ${[5, 0]}                                  | ${"-315° angle at (0, 0)"}
+        ${[5, 0]}   | ${[0, 0]} | ${[-5, -5]}                                | ${"135° angle at (0, 0)"}
+        ${[5, 0]}   | ${[0, 0]} | ${[5, -5]}                                 | ${"45° angle at (0, 0)"}
+        ${[0, 5]}   | ${[0, 0]} | ${[5, 0]}                                  | ${"90° angle at (0, 0)"}
+        ${[-5, 5]}  | ${[0, 0]} | ${[5, 0]}                                  | ${"135° angle at (0, 0)"}
+        ${[-5, -5]} | ${[0, 0]} | ${[5, 0]}                                  | ${"135° angle at (0, 0)"}
+        ${[5, -5]}  | ${[0, 0]} | ${[5, 0]}                                  | ${"45° angle at (0, 0)"}
```

Comment:
Question: This change is because reflexive angles are not allowed by default now?

Reply:
They always weren't allowed by default ... technically. Previously, when you moved the bottom arm, you could accidentally generate reflexive angles regardless of the value of the `allowReflexAngles` property. With the previous validation scoring, these angles would sometimes calculate as negative angles. Generating negative angles was a major sign of this historical bug and should never have been possible. 😅

Now that we're calculating these angles correctly, we're always being provided positive values. :)